superset-frontend/package.json

```diff
@@ -61,14 +61,14 @@
     "@emotion/core": "^10.0.28",
     "@superset-ui/chart": "^0.14.1",
     "@superset-ui/chart-composition": "^0.14.1",
-    "@superset-ui/color": "^0.14.2",
+    "@superset-ui/chart-controls": "^0.14.1",
+    "@superset-ui/color": "^0.14.1",
```

Comment:
were the downgrades here intentional?

Reply:
@etr2460  I didn't clean up conflict correctly. All package should be up to date now.